src/vs/code/test/electron-main/nativeHelpers.test.ts

```diff
@@ -30,7 +30,17 @@ suite('Windows Native Helpers', () => {
 	test('vscode-windows-ca-certs', async () => {
 		// @ts-ignore Windows only
 		const windowsCerts = await import('vscode-windows-ca-certs');
+		const store = new windowsCerts.Crypt32();
 		assert.ok(windowsCerts, 'Unable to load vscode-windows-ca-certs dependency.');
+		let certCount = 0;
+		try {
+			while (store.next()) {
+				certCount++;
+			}
+		} finally {
+			store.done();
+		}
+		assert(certCount > 0);
```

Comment:
On a fresh Windows machine the cert store might have very few certificates because Edge downloads and adds to it on demand. Not sure if it can be empty, but if we see this test failing we need to consider that possibility.